operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ConfigurationServiceOverrider.java

```diff
@@ -188,6 +190,12 @@ public ConfigurationServiceOverrider withCloneSecondaryResourcesWhenGettingFromC
     return this;
   }
 
+  public ConfigurationServiceOverrider previousAnnotationUsageBlocklist(
+      List<Class<? extends HasMetadata>> previousAnnotationUsageBlacklist) {
+    this.previousAnnotationUsageBlocklist = previousAnnotationUsageBlacklist;
```

Comment:
See the previous comment on how this should probably be adding to the default set rather than completely replace it.

Reply:
Responded above.